tensornetwork/contractors/greedy_contractor_test.py

```diff
@@ -19,6 +19,8 @@
 import numpy as np
 from tensornetwork.contractors import greedy_contractor
 from tensornetwork import network
+import tensorflow as tf
+tf.enable_v2_behavior()
```

Comment:
This isn't required anymore after https://github.com/google/TensorNetwork/commit/113015b9ff034d38432e30a663f048067f25c6de